test/test.dart

```diff
@@ -241,20 +233,28 @@ String generateRandomString(int len) {
 }
 
 Realm getRealm(Configuration config) {
+  if (config is FlexibleSyncConfiguration) {
+    config.sessionStopPolicy = SessionStopPolicy.immediately;
+  }
+
   final realm = Realm(config);
   _openRealms.add(realm);
   return realm;
 }
 
-Future<void> tryDeleteFile(FileSystemEntity fileEntity, {bool recursive = false}) async {
-  for (var i = 0; i < 20; i++) {
+Future<void> tryDeleteRealm(String path) async {
+  for (var i = 0; i < 100; i++) {
     try {
-      await fileEntity.delete(recursive: recursive);
-      break;
+      Realm.deleteRealm(path);
+      await File('$path.lock').delete();
+      return;
     } catch (e) {
+      print('Failed to delete realm at path $path. Trying again in 50ms');
       await Future<void>.delayed(const Duration(milliseconds: 50));
```

Comment:
Maybe complain a bit. Otherwise this will just hide problems I think?

Reply:
How can we complain?